codespell_lib/data/dictionary.txt

```diff
@@ -20409,6 +20420,13 @@ negatiotiators->negotiators
 negativ->negative
 negatve->negative
 negible->negligible
+negitiable->negotiable
+negitiate->negotiate
+negitiated->negotiated
+negitiates->negotiates
+negitiating->negotiating
+negitiation->negotiation
+negitiations->negotiations
```

Comment:
```suggestion
negitiator->negotiator
negitiators->negotiators
```

Reply:
Oops! So lines 20429 and 20430 were duplicate, weren't they? Don't CI checks catch this?

Reply:
Everything looks fine now. Either GitHub displayed the diff in a funny way yesterday, or I got confused when reading the diff.

Reply:
That's good, but `negitiations->negotiations` still wants adding please!

Reply:
I've added a new suggestion for it.